src/Maestro/Maestro.Authentication/AuthenticationConfiguration.cs

```diff
@@ -111,6 +111,12 @@ public static void ConfigureAuthServices(this IServiceCollection services, IConf
                             || context.User.IsInRole(prodconSvcsRole);
                     });
                 });
+                options.AddPolicy("RequireAdminAccess", policy =>
```

Comment:
Can you pull this name up with the rest of the policy names?

Reply:
Sure thing